app/src/Search/SearchController.php

```diff
@@ -26,7 +25,13 @@ class SearchController extends BaseController
     /**
      * @var integer The number of search results to show per page
      */
-    protected $itemsPerPage = 10;
+    protected $itemsPerPage;
+
+    public function __construct(Slim $app)
```

Comment:
If we use ```Slim``` here there should also be a ```use Slim\Slim;``` in this file, even though it's already set in the BaseController-File. Makes it more readable IMHO